### PR TITLE
REAL MEN CANNOT BE DISARMED OF THEIR SLAPS

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: slapping 50 times NOW MAKES YOU A REAL MAN
add: 150 slaps makes you A GOD AMONG MEN!
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -607,6 +608,16 @@
...
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold danger'>YOU HAVE ASCENDED PAST MANLIHOOD, INTO GODHOOD!</span>")
+				icon_state = "disintegrate"
+				item_state = "disintegrate"
+				lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+				righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```
Comment (ShizCalev) [inline code/game/objects/items/weaponry.dm]: it already inherits this var.

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -607,6 +608,16 @@
...
  		"You hear a slap.")
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold danger'>YOU HAVE ASCENDED PAST MANLIHOOD, INTO GODHOOD!</span>")
+				icon_state = "disintegrate"
+				item_state = "disintegrate"
+				lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
```
Comment (ShizCalev) [inline code/game/objects/items/weaponry.dm]: it already inherits this var.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -607,6 +608,16 @@
...
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && icon_state != "disintegrate")
+			to_chat(user, "<span class='bold danger'>YOU HAVE ASCENDED PAST MANLIHOOD, INTO GODHOOD!</span>")
+			icon_state = "disintegrate"
+			item_state = "disintegrate"
+			lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+			righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```
Comment (ShizCalev) [inline code/game/objects/items/weaponry.dm]: it already inherits this var.

File: code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -607,6 +608,16 @@
...
  		"You hear a slap.")
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && icon_state != "disintegrate")
+			to_chat(user, "<span class='bold danger'>YOU HAVE ASCENDED PAST MANLIHOOD, INTO GODHOOD!</span>")
+			icon_state = "disintegrate"
+			item_state = "disintegrate"
+			lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
```
Comment (ShizCalev) [inline code/game/objects/items/weaponry.dm]: it already inherits this var.

**Review by ShizCalev — DISMISSED** [review]

**Comment by praisenarsie** [comment]
too easy, make it 250 and 750

File: code/game/objects/items/weaponry.dm [hunk 5]
```diff
@@ -607,6 +608,16 @@
 		user.visible_message("<span class='danger'>[user] slaps [M] in the [(aim_for_face)?"face":user.zone_selected]!</span>",
  		"<span class='notice'>You slap [M] in the [(aim_for_face)?"face":user.zone_selected]! </span>",\
  		"You hear a slap.")
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && icon_state != "disintegrate")
```
Comment (Cyberboss) [inline code/game/objects/items/weaponry.dm]: No icon_state logic checks

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm [hunk 6]
```diff
@@ -607,6 +608,16 @@
...
  		"<span class='notice'>You slap [M] in the [(aim_for_face)?"face":user.zone_selected]! </span>",\
  		"You hear a slap.")
+		slaps++
+		if(slaps > 50 && !(flags_1 & NODROP_1))
+			to_chat(user, "<span class='bold notice'>Congrats, you are now a REAL MAN! Nobody can disarm you of your slapping abilities now!</span>")
+			flags_1 |= NODROP_1
+		if(slaps > 150 && icon_state != "disintegrate")
+			to_chat(user, "<span class='bold danger'>YOU HAVE ASCENDED PAST MANLIHOOD, INTO GODHOOD!</span>")
+			icon_state = "disintegrate"
+			item_state = "disintegrate"
```
Comment (Cyberboss) [inline code/game/objects/items/weaponry.dm]: update_icon

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm [hunk 7]
```diff
@@ -609,19 +610,26 @@
...
 			desc = "You are a god among men, slapper!"
+			ascended = TRUE
+			update_icon()
 		return
 	else
 		..()
 
+/obj/item/slapper/update_icon()
+	. = ..()
+	if(slaps >= 150 && !ascended)
```
Comment (Cyberboss) [inline code/game/objects/items/weaponry.dm]: no need to check ascended jerte

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm (hunk 7 above)
Comment (Qustinnus) [reply code/game/objects/items/weaponry.dm]: are you okay buddy

Comment (MoreRobustThanYou) [reply code/game/objects/items/weaponry.dm]: he's probably drunk tbh

**Comment by deathride58** [comment]
Speedmerge this shit what are you all doing

**Comment by YakumoChen** [comment]
please don't make it more annoying then it currently is

**Comment by AnturK** [comment]
No.

**Comment by optimumtact** [comment]
thank you

**Pull request closed without merging**